### DISCUSSION TITLE
五月更新 - 1

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1136] [通知]
## 更改记录

提前祝大家五一开心，不要因为调休就破坏了自己的好心情，毕竟你还有哔哩 (/ω＼)

这是一次修复更新，主要的功能更新列表在 [五月更新 - 预](https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2205.0.0) 中，高亮功能如下：

- 支持漫游（低调使用，服务器自己找）
- 弹幕引擎升级为 [寒霜弹幕使](https://github.com/cotaku/DanmakuFrostMaster)，与官方app的弹幕体验一致
- 支持综合动态
- 支持新动态通知推送

## 🐛 Bug 修复

- 修复快捷键没有正常切换迷你视图的问题 @Richasy (#1133)
- 修复断网自动退出及登录流程问题 @Richasy (#1132)
- 修复始终弹出同一个动态的问题 @Richasy (#1120)
- 修复部分动态类型评论打不开的问题 @Richasy (#1119)
- 限制漫游服务器文本框宽度 @Richasy (#1118)

*接下来**业余时间**的工作重心将转移至 WFA，哔哩的更新不会那么频繁了*

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2205.1.0'>五月更新 - 1</a>.</em>

**Comment by msycn** [comment]
感谢分享，R大五一快乐！

**Comment by ghost** [comment]
新版本看视频的时候弹幕一卡一卡的

**Reply by mohiko** [reply]
可以用漫游好评

**Reply by mohiko** [reply]
一个能用的代理服务器都没有，这哔哩漫游的功能就是个摆设啊

**Comment by 1336841591** [comment]
弹幕如果不放全屏显示重叠的太厉害，根本看不清，有点苦恼。
滚动体验还是有些卡卡的
此外还卡退了一次，也不知道是什么原因

**Reply by 1336841591** [reply]
弹幕确实会卡顿

**Reply by ghost** [reply]
更新之后，好像我固定的UP也没了

**Reply by Richasy** [reply]
在弹幕设置里开启 `弹幕合并` 与 `自动限制同屏弹幕数` 后，重叠情况会有缓解，同时也可以调整弹幕透明度以降低遮挡。

弹幕卡顿的问题修了，下次更新

**Reply by Richasy** [reply]
> 更新之后，好像我固定的UP也没了

@ConfusedOneself 功能更新后把固定内容做了整合，需要重新固定一下

**Reply by 1336841591** [reply]
那这个弹幕遮挡下个版本会修嘛，我感觉这两项都开启后效果还是一般。因为看视频为避免影响观感和遮挡内容，我一般开弹幕显示区域为1/4屏或1/2屏，但是这种情况在此版本中弹幕重叠还是挺严重的，就让我这种弹幕和视频同事看的人很难受，望大佬解惑🫡

**Comment by dadada0119** [comment]
感觉寒霜弹幕使的弹幕效果没有以前的好看。。

**Comment by cesaryuan** [comment]
WFA是什么东东

**Reply by Mikancake** [reply]
Warframe Alerting Prime，作者的另一个作品

**Comment by happylittle2010** [comment]
漫游是什么

**Comment by zhidurensheng** [comment]
匹配下哔哩.plus，作者说你不匹配

**Comment by mumuxiyu** [comment]
好几个版本过去了，收藏里还是没有全部播放

**Comment by ghost** [comment]
同样的感觉

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
发件人: dadada0119 ***@***.***>
发送时间: 2022年4月27日星期三 下午10:18
收件人: Richasy/Bili.Uwp ***@***.***>
抄送: ConfusedOneself ***@***.***>; Mention ***@***.***>
主题: Re: [Richasy/Bili.Uwp] 五月更新 - 1 (Discussion #1136)


感觉寒霜弹幕使的弹幕效果没有以前的好看。。

―
